pymc4/distributions/continuous.py

```diff
@@ -81,33 +80,27 @@ class Normal(ContinuousDistribution):
 
     Parameters
     ----------
-    mu : float|tensor
-        Mean.
-    sigma : float|tensor
-        Standard deviation (sigma > 0).
+    loc : float
+        Location parameter. For the normal distribution, this is the mean.
+    scale : float
+        Scale parameter. For the normal distribution, this is the standard
+        deviation (scale > 0).
 
     Examples
     --------
     .. code-block:: python
         @pm.model
         def model():
-            x = pm.Normal('x', mu=0, sigma=10)
-
-    Developer Notes
-    ---------------
-    Parameter mappings to TensorFlow Probability are as follows:
-
-    - mu: loc
-    - sigma: scale
+            x = pm.Normal('x', loc=0, scale=10)
     """
 
-    def __init__(self, name, mu, sigma, **kwargs):
```

Comment:
Now that we're just forwarding can't we move all this up into the base class and remove it here?

Reply:
I think @ColCarroll was opposed, for what I think is a pretty good reason: tab-completion and docstring-finders in many IDEs might rely on having explicit `__init__`s.

If we just had `*args, **kwargs`, then it would be hard for new users to find out the required parameters by e.g. running `pm.distributions.Normal??` in IPython.